builder/nginx.conf

```diff
@@ -53,27 +53,5 @@ http {
     default ".pb";
   }
```

Comment:
The 3 option blocks above are very important in making the registry work, and now they are somewhat disconnected (the `$prefix` and `$suffix` variables are created here but referenced in `default.conf`). Can we connect them in some more obvious way?

Reply:
I don't think so. The map can only be in the `http` context.